src/components/SchedulePage/Schedule.js

```diff
@@ -5,28 +5,40 @@ import Typography from '@material-ui/core/Typography';
 import Container from '@material-ui/core/Container';
 import List from '@material-ui/core/List';
 import Divider from '@material-ui/core/Divider';
+
 import StickyTimeSlot from '../SchedulePage/StickyTimeSlot';
 import workshopSchedule from '../../data/WorkshopSchedule';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
 	title: {
 		fontWeight: 'bold',
-		paddingTop: 24,
-		paddingBottom: 16
+		paddingTop: theme.spacing(11),
+		paddingBottom: theme.spacing(4)
+	},
+	divider: {
+		height: '2px',
+		marginLeft: theme.spacing(4),
+		marginRight: theme.spacing(4),
+		marginBottom: theme.spacing(7),
+		[theme.breakpoints.up('sm')]: {
+			marginLeft: theme.spacing(7),
+			marginRight: theme.spacing(7),
+			marginBottom: theme.spacing(9)
+		}
 	}
-});
+}));
 
 const SchedulePage = () => {
 	const classes = useStyles();
 	return <Container maxWidth='md'>
-		<Typography align='center' component='h1' variant='h4' className={classes.title}>Schedule</Typography>
+		<Typography align='left' component='h1' variant='h4' className={classes.title}>Schedule</Typography>
 		<List>
-			{workshopSchedule.map(timeslot => {
+			{workshopSchedule.map((timeslot, index) => {
 				return (
 					<React.Fragment key={timeslot.startTime}>
-						<Divider component='li' />
 						<li>
 							<StickyTimeSlot events={timeslot.events} time={timeslot.startTime} />
+							{index === workshopSchedule.length - 1 ? null : <Divider className={classes.divider} />}
```

Comment:
Why change this to be different from what we had? I.e., `<Divider component='li' />` instead of putting Divider in another li.